sdk/core/core-paging/samples-dev/getPagedAsyncIteratorSample.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
```

Comment:
This feels more like a test than a sample.

Reply:
Good point, I converted it to tests and kept only the for-await looping in the sample. Not entirely happy with the definition of the operation itself so any suggestions there are welcome.